extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1076,17 +1076,23 @@ export class Bind {
 
     switch (property) {
       case 'text':
-        element.textContent = String(newValue);
-        // If this is a <title> element in the <head>, update document title.
+        let updateTextContent = true;
+        const stringValue = String(newValue);
+
+        // textContent on <textarea> only works before interaction.
+        if (tag == 'TEXTAREA') {
+          element.value = stringValue;
+          // Don't also update textContent to avoid disrupting focus.
```

Comment:
nit. Just "Don't update textContent to avoid..."

Reply:
Could this use a `break` to exit early instead of creating and tracking `updateTextContent`?

```javascript
case 'text':
  const stringValue = String(newValue);

  // textContent on <textarea> only works before interaction.
  if (tag === 'TEXTAREA') {
    element.value = stringValue;
    break;
  }
  // If <title> element in the <head>, also update the document title.
  if (tag === 'TITLE'
      && element.parentNode === this.localWin_.document.head) {
    this.localWin_.document.title = stringValue;
  }

  element.textContent = stringValue;
  break;
```

Here's an example of the pattern outside this usage, where calling `foo('foo')` will exit the case early and avoid console logging 'exit', but `foo('bar')` will output both 'enter' and 'exit'.

```javascript
function foo(string) {
  switch (string) {
    case 'foo':
      console.log('enter');
      if (string === 'foo') {
        console.log('conditional start');
        break;
      }
      console.log('exit');
      break;
    case 'bar':
      console.log('enter');
      if (string === 'foo') {
        console.log('conditional start');
        break;
      }
      console.log('exit');
      break;
    default:
      console.log('default');
  }
}
```

Reply:
Hmm that's a bit cleaner, thanks.

Reply:
Finally got CI to pass after many flakes, going to merge first.